### ISSUE TITLE
Stuck in "Confirming Match" and being punished for leaving

### ISSUE THREAD

**Issue by calebphess** [ValveSoftware/Dota2-Gameplay#11791]
### Description

You guys should really add a time counter for matchmaking that allows you to cancel the game without punishment if the match doesn't start X minutes from when a matchmaking group is created. E.G. Stop punishing us for your bugs. 

### Example Match ID (and possibly Timestamp)

DNE

**Comment by AlphaBcash** [comment]
I agree this is ridiculous, the timer for my confirmed match kept restarting and when i quit dota i was punished

**Comment by tblamps** [comment]
me as well, i waited for 5 minutes before canceling, getting a 3 minute cooldown. q'd again after restarting dota and immediately popped into a confirming match, currently at 2 minutes. NA immortal queue

**Comment by tblamps** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143454458/c1538fcd-df17-4b46-aafb-7a63efead17a)

**Comment by tblamps** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143454458/d3a537a5-45a4-47a2-8c85-1ec5bc908ed9)

**Comment by xbxb30** [comment]
It's really annoying, this bug no longer has a solution, it's too annoying, they penalize me for 15 minutes when it remains bugged, can't you fix that problem? It has zzzz months, is this how they celebrate 10 years of Dota 2?

**Comment by hurricanechang** [comment]
The issue seems to be worst when there is more people in a party for me. Two people it occurs once a day but when its 3 or more, it happens too frequently. Needs fixing asap!

**Comment by nerdpope** [comment]
I don't have anyone in my friends list, so it must be server load related. It's Sunday, I assume the weekend person is trying to reach the primary team, I hope.

**Comment by stelonix** [comment]
No party and haven't been to a party for MONTHS. I play solo and it was just fine last night. Now it's stuck on confirming match and if I cancel I'll be HOURS banned from playing.

It's really really annoying to get punished for your bugs and at the same time not getting a single response about it.

**Comment by targaro9107** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/131053883/d96ff28e-5921-445b-b223-52dc6e12e86b)

**Comment by xbxb30** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/116115480/f89e7bea-1e25-490b-8c7d-e0ad87d109ea)

**Comment by Demon7101990** [comment]
I currently have a 24hr ban because of this bullshit.

**Comment by pedSoares** [comment]
no one is playing..
10 year celebration, enjoy it.

**Comment by LuisAguilarDev** [comment]
24H penalty for a bug nice :)

**Comment by jjpaulo2** [comment]
Same problem in South America (Brasil, Argentina, Chile).

**Comment by EatPlayLog** [comment]
Same here, and I even restarted my router, my PC as well as reinstalled the game, which I could do in less than the duration of my last ban, which was 30 minutes. Now 45 minutes.

The most aggravating part: reporting the bug via Steam and asked to attach a screen shot.

A screen shot of what exactly...? Nothing happening?

**Comment by erickfiori** [comment]
Same thing here. I've tried all possible troubleshooting but still 'Confirmg match'.

**Comment by FreitasVitor** [comment]
the servers are down people

**Comment by abs1034** [comment]
Same. This crap has been happening all weekend and we are being punished for their crappy servers. Not to mention the wonderful all-player disconnect that keeps happening when you finally get in a match. Can't play and yet I'm paying for this??

**Comment by skulltom666** [comment]
same problem here, waits until Tuesday developers maybe wake up and do their job. and every 5min respawn cd again 
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/106767254/9d75a5dc-5a43-462a-a1cf-4aefc8e98184)

**Comment by NoKGamer** [comment]
I've been sitting here for 20 minutes while it hits 2-4 minutes and then restarts the timer.. I guess valve doesn't want me to play anymore
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/37144602/9e161483-1149-42ed-b52e-144536bd2bfb)

**Comment by thearctican** [comment]
This is a feature request. Your filing is not a bug.

**Comment by ajaykauthale1** [comment]
Also, you cannot watch other match while confirming

**Comment by ajaykauthale1** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/22283064/448deb45-bf9d-4037-9c1a-90dec9c45675)

**Comment by Chrisjames416** [comment]
yikes. this outage sucks. 
just letting you all know after all is said and done, i will still subscribe to DOTA+

**Comment by gachalifegame** [comment]
I am extremely intrigued in this subject. Even though I have read numerous articles, I believe yours is the best. [capybara clicker](https://capybaragame.io/)
